messaging/src/main/java/org/axonframework/eventhandling/pooled/Coordinator.java

```diff
@@ -819,11 +828,12 @@ private Map<Segment, TrackingToken> claimNewSegments() {
                                 () -> tokenStore.fetchToken(name, segment)
                         );
                         newClaims.put(segment, token);
+                        logger.debug("Processor [{}] claimed the token for segment {}.", name, segmentId);
```

Comment:
I would argue that this is valuable information and would prefer it to be on INFO level

Reply:
I agree. It likely should not be logged too often, either.

Reply:
Sure thing guys!